slsDetectorServers/mythen3DetectorServer/slsDetectorServer_defs.h

```diff
@@ -20,9 +20,12 @@
 #define DEFAULT_EXPTIME				(100*1000*1000)		//ns
```

Comment:
You can remove #include "RegisterDefs.h" from the beginning of this file, as we don't use any constants from this file yet.